### PR TITLE
Add stats to home page

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Add the stats section to the home page.

Also, add a few fixes here and there:
* Add `external` prop to links
* Add blue theme to Title
* Fix font size on Stats

### CODE REVIEW

File: src/components/Themes/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export default {
   red: "redTheme",
+  blue: "blueTheme",
   neonCarrot: "neonCarrotTheme",
   greyishBlue: "greyishBlueTheme",
   lightBlue: "lightBlueTheme",
```
Comment (DavideSilva) [inline src/components/Themes/index.js:5]: just my personal preference, and feel free to disregard but can we sort this alphabetically?

File: src/components/Title/index.js [hunk 2]
```diff
@@ -18,8 +18,12 @@ Title.defaultProps = {
...
 Title.propTypes = {
   children: PropTypes.node.isRequired,
-  theme: PropTypes.oneOf([Themes.red, Themes.neonCarrot, Themes.greyishBlue])
-    .isRequired,
+  theme: PropTypes.oneOf([
+    Themes.red,
+    Themes.blue,
+    Themes.neonCarrot,
+    Themes.greyishBlue,
+  ]).isRequired,
```
Comment (DavideSilva) [inline src/components/Title/index.js:26]: again, just my opinion, but we could try to keep this sorted

File: src/sections/HomeStats/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Section from "root/components/Section";
+import Themes from "root/components/Themes";
+import Stats from "root/components/Stats";
+import Title from "root/components/Title";
+import BigText from "root/components/BigText";
+import Link from "root/components/Link";
+
```
Comment (DavideSilva) [inline src/sections/HomeStats/index.js:10]: IMHO, we could sort the imports. As with the others comments about sorting, feel free to disregard this

**Review by DavideSilva — APPROVED** [review]
Left some comments about sorting things. Feel free to address those, or don't, but this is good to go :shipit:

File: src/sections/HomeStats/index.js (hunk 3 above)
Comment (jfranciscosousa) [reply src/sections/HomeStats/index.js:10]: Yes

File: src/components/Title/index.js (hunk 2 above)
Comment (jfranciscosousa) [reply src/components/Title/index.js:26]: Yes

File: src/components/Themes/index.js (hunk 1 above)
Comment (jfranciscosousa) [reply src/components/Themes/index.js:5]: Yes

**Pull request merged**